test/example.test.js

```diff
@@ -1,18 +1,34 @@
 // IMPORT MODULES under test here:
-// import { example } from '../example.js';
+// import { findFriendByName } from '../data-utils.js';
+import { renderMushroom } from '../render-utils.js';
 
 const test = QUnit.test;
 
-test('time to test a function', (expect) => {
+test('renderMushroom function returns a div element with the class of mushroom', (expect) => {
     //Arrange
     // Set up your arguments and expectations
-    const expected = true;
+    const expected = `<div class="mushroom"></div>`;
     
     //Act 
     // Call the function you're testing and set the result to a const
-    const actual = true;
+    const actual = renderMushroom();
 
     //Expect
     // Make assertions about what is expected versus the actual result
-    expect.equal(actual, expected);
+    expect.equal(actual.outerHTML, expected, 'returns a div element with the class of mushroom: <div class="mushroom"></div>');
 });
+
+
+// test('findFriendByName function returns a friend with the correct name', (expect) => {
+//     //Arrange
+//     // Set up your arguments and expectations
+//     const expected = 'Peter';
+    
+//     //Act 
+//     // Call the function you're testing and set the result to a const
+//     const actual = findFriendByName('Peter', 'let friendsArr = ['May', 'Jeff', 'Daisy', 'Peter', 'Amelia', 'Tom']');
```

Comment:
I don't think you can declare a variable as a function argument.  I would define your friendsArr as a variable on a previous line and then pass that variable into your function call as an argument.